### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1126] [Galaxy Trivia] [answered]
```
14  91  62  53  64  96 ?
```

**Comment by CelioAmaral** [comment, marked as answer]
Answer
48.

These are the squares of numbers without the correct spacing.

   1 4 9 16 25 36 49 64 81 100

with no spaces becomes:

   149162536496481100

broken into blocks of two:

   14 91 62 53 64 96 48 11 00

So the next two in the sequence are 48.
